actors/builtin/miner/miner_actor.go

```diff
@@ -264,8 +267,6 @@ func (a *Actor) PreCommitSector(rt Runtime, params *PreCommitSectorParams) *adt.
 	depositReq := precommitDeposit(st.getSectorSize(), params.info.Expiration-rt.CurrEpoch())
 	confirmPaymentAndRefundChange(rt, depositReq)
 
-	// TODO HS Check on valid SealEpoch
```

Comment:
Remove from the Precommit: this is done in `verifySeal` after the commit is done. Functionally the same